ProcessQueue/__init__.py

```diff
@@ -248,8 +267,14 @@ def main(msg: func.QueueMessage):
                 
                 # If we are not on the first row:
                 if name!=value:
-                    print(result)
-                    print("Frequency",frequency)
+                    #Get the field_id
+                    #If more than one field with the same name, pick the one in the current sheet
+                    field_id=get_field_id(namexids,field=name)
+                    if len(field_id)>1:
+                        field_id=field_id[idxsheet-2]

```

Comment:
Will this logic work correctly if some but not all of the sheets contain the same field name? So if we have 3 tables and table 1 and table 3 contain `my_field_name` then `get_field_name` will return 1 list of length 2, which wouldn't then match the `idxsheet-2` index used.

Reply:
ah yes you're right I'll try to come up with a better solution